rnotes/runner.py

```diff
@@ -402,12 +410,25 @@ def branch_check(self):
             print("Check merge target:", target)
             diff_base = target
 
-        diff = self.git("diff", "--name-only", "--diff-filter=A", diff_base)
+        need_notes = False
+        all_diff = self.git("diff", "--name-only", diff_base)
+        for ent in all_diff.split("\n"):
+            ent = ent.strip()
+            if not ent or self.not_important(ent):
+                continue
+            if ent.startswith(self.notes_dir):
```

Comment:
Should this skip deleted files? Should be rare in practice, but I could see a use case if some bad notes made it into trunk and need to be renamed/consolidated/rewritten.

Reply:
yep

Reply:
can just skip fnf errs, should be enough